amplify/backend/ts-code/src/api/AddItem.ts

```diff
@@ -107,6 +107,7 @@ export class AddItem {
 interface ScratchInterface {
     id?: string,
     name?: string,
+    displayName?:string,
```

Comment:
We don't want an additional input in the `ScratchInterface`. Essentially, we want to pass in name (where the user can pass a uppercase name like "iPhone"), and then figure out in the `MainTable` how to properly populate the `name` and `displayName` fields respectively after that.